docs/components/AppLayout.js

```diff
@@ -32,7 +34,7 @@ export default function AppLayout({ children }: Props): Node {
             lgDisplay="flex"
             justifyContent="center"
           >
-            <Box width="100%" maxWidth={1544}>
+            <Box width="100%" data-test-id="FOO" maxWidth={CONTENT_MAX_WIDTH_PX}>
```

Comment:
Why do we have `data-test-id="FOO"`?

Reply:
🤦 
https://github.com/pinterest/gestalt/pull/1861